.github/pull_request_template.md

```diff
@@ -31,30 +31,30 @@ Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)
 
 - [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
 - [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
-- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
+- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``rapid7/insightconnect-python-3-38-slim-plugin:4`` and ``rapid7/insightconnect-python-3-38-plugin:4``
```

Comment:
Should we rephrase this as we don't recommend slim usually?

Reply:
🤔💭
Will we leave this new change as is and come back to amend it once we have this slim package confusion sorted out?

Reply:
I'd be inclined to handle this the other way around unless we can be sure the sizing of the slim package is reduced.